graphics/scripts/globals.js

```diff
@@ -0,0 +1 @@
+const DASHBOARD_BUNDLE_NAME = 'ipl-overlay-controls';
```

Comment:
This is only used in the replicants file, as far as I know - It's probably better not to have a separate file for this

Reply:
I disagree, as this is how Low Ink, SOS also handles this variable, it's better if we keep it consistent across the board